### ISSUE TITLE
Zul'Gurub - High Priest Thekal (Zulian tiger boss)

### ISSUE THREAD

**Issue by Turanius** [SunwellTracker/issues#1634]
**Decription:**
High priest thekal doesn't phase.

**How it works:**
You kill the boss and the NPC(Adds) at the same time and nothing happens, doesn't go to phase 2 where he respawns, thus making him unkillable.

**How it should work:**
When you kill both adds and boss at the same time, it phases into phase 2 where the boss respawns, when you kill him in his "final form" he the encounter ends, then you gain access to his loot-table.

**Source (you should point out proofs of your report, please give us some source):**
https://www.youtube.com/watch?v=OHsF3kuX0PY
a video of a hunter soloing the instance, how it should work, this encounter.

**Comment by Kokaina** [comment]
Fixed, will be implemented with next update.

**Comment by Platinumcoated** [comment]
As of today, had this bug twice already. Previous ID we tried doing the boss and it bugged as described, We didn't think much of it and moved on. However today, it happened again, so we made a ticket. GM reset the encounter for us and after doing phase one, killing them at the same time, it bugged in the same way. Every mob dead, but no tiger boss.
The body of the High Priest is actually on the ground, sparkling as if it had loot in it, however it cannot be clicked, so no loot can be obtained.
I've done this boss plenty of times before and never had any issues, only previous ID (few days ago) and today.
